### PR TITLE
Add Ubuntu Bionic support/testing

### PR DESCRIPTION
**Pull request by odyssey4me**
In order to support and to validate support for
Ubuntu 18.04 (Bionic), we add the appropriate
molecule configuration to test it, along with
the only change necessary for the tests to pass.

### CODE REVIEW

**Comment by odyssey4me** [comment]
Odd, the image referenced is public so it should work. I'm entirely new at using docker though, so any pointers which might help are welcome. :)

File: molecule.yml [hunk 1]
```diff
@@ -17,6 +17,12 @@ docker:
       privileged: true
       volume_mounts:
         - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    - name: bionic
+      image: odyssey4me/ubuntu-molecule
```
Comment (odyssey4me) [inline molecule.yml]: I'll adjust this once https://github.com/paulfantom/dockerfiles/pull/1 merges and the image is published.

Comment (paulfantom) [reply molecule.yml]: PR is merged and images are public :tada:

**Comment by paulfantom** [comment]
This is great! Just change image name and I'll merge it.

File: README.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ We provide demo site for full monitoring solution based on prometheus and grafan
 The preferred way of locally testing the role is to use Docker and [molecule](https://github.com/metacloud/molecule) (v1.25). You will have to install Docker on your system. See Get started for a Docker package suitable to for your system.
 All packages you need to can be specified in one line:
 ```sh
-pip install ansible ansible-lint>=3.4.15 molecule==1.25.0 docker testinfra>=1.7.0
+pip install ansible 'ansible-lint>=3.4.15' 'molecule==1.25.0' docker git-semver 'testinfra>=1.7.0,<=1.10.1'
```
Comment (paulfantom) [inline README.md]: One thing, `git-semver` isn't needed for local testing, please remove it from this line.

**Comment by paulfantom** [comment]
And about issue with travis not downloading your docker image:
We are using a simple script to prefetch docker images, so molecule won't build them in every CI pipeline. This script is located in `.travis/imges.sh` and needs to have every image used by molecule.

File: README.md (hunk 2 above)
Comment (odyssey4me) [reply README.md]: Done.

File: molecule.yml (hunk 1 above)
Comment (odyssey4me) [reply molecule.yml]: Awesome, thanks!

**Comment by odyssey4me** [comment]
Thanks for the tip about the images script - I missed that!

File: README.md (hunk 2 above)
Comment (odyssey4me) [reply README.md]: Done.

**Comment by paulfantom** [comment]
Thanks!

**Pull request merged**